common/model-views.cpp

```diff
@@ -4166,11 +4170,58 @@ namespace rs2
         }
         return clicked;
     }
-    bool device_model::prompt_toggle_advanced_mode(bool enable_advanced_mode, const std::string& message_text, std::vector<std::string>& restarting_device_info, viewer_model& view, ux_window& window)
+
+    // Create a process windows with process details from the caller,
```

Comment:
"process windows"? why process_model for the name of the function?

It's a message dialog with a separate status, no? Why not name it similarly to the yes_no_dialog?

Reply:
Cause it is not really a dialog, it is more related to display a process
I use yes_no_dialog for the first window, and a process window for the process itself

Reply:
What makes it a process window?
Because it has statuses?
Can you do anything else in the app while this window is up? If not, it's a modal dialog.
"status_dialog"?

Reply:
Changed